README.md

```diff
@@ -38,6 +57,24 @@ Notes:\\
 $I:=D^1$ is the interval used for the definition of a homotopy.
 
 \subsubsection*{Disjoint union}
+The Disjoint Union is most properly described in terms of universal properties,
+but because of that it has many equivalent concrete descriptions.  So we pick
+a totally arbitrary definition, the Disjoint Union of a collection of sets,
+$X_i$, indexed by $i\in I$, is just $\bigcup_{i\in I} \bigcup_{x\in X_i} (i, x)$.
```

Comment:
Conditions on I?

Reply:
I don't think there are any conditions required for I.

Reply:
Countability? What's confusing is that I is also used for the interval in the text around it.